help/status-and-availability.md

```diff
@@ -50,10 +50,34 @@ You can set a status emoji, status message, or both.
 Status emoji are shown next to a user's name in the sidebars, message feed,
 and compose box.
 
-You can view status messages by hovering over your or anyone else's name in the
-left or right sidebar, or by clicking the user's name or avatar in the main
-message feed to open their **user card**. If someone hasn't set a message as
-part of their status, then no status message will appear.
+{start_tabs}
+
+{tab|desktop-web}
+
+1. Hover over a user's name in the right sidebar.
+
+1. Click on the **ellipsis** (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>)
+   to the right of their name to open their **user card**. You can also click on
+   a user's profile picture or name on a message they sent.
```

Comment:
These instructions aren't right. If you hover over the name in the name in the left or right sidebar (is there a reason you dropped the left sidebar option?), you get the status message in the tooltip. You don't need to click on anything.

Reply:
Opening a user card is an alternative way to view their status message.

Reply:
Oh, I was confused about that. I think we should add that users should look in the **Direct messages** section in the left sidebar since it is collapsible.